File: pyg_shop_web/src/main/webapp/js/service/statisticService.js
app.service('statisticService',function ($http) {
    //读取列表数据绑定到表单中
    this.findAll=function(){
        return $http.get('../statistic/findAll.do');
    }
    this.findOne=function (id) {
        return $http.get('../statistic/findOne.do?id='+id);
    }
    this.update=function (entity) {
        return $http.post('../statistic/update.do',entity);
    }
})

File: pyg_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/StatisticService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbItem;
import entity.Result;

import java.util.List;

public interface StatisticService {
    public List<TbItem> findAll(String sellerId);
    public TbItem findOne(Long id);
    public Result updateItem(TbItem tbItem);
}
